README.md

```diff
@@ -7,11 +7,11 @@ MarkLogic Data Hub is a data integration platform and toolset that helps you qui
 
 # Version Support
 
-For Data Hub version 5.0, you need:
+For Data Hub version 5.2, you need:
 
-  - MarkLogic 9.0-10 or later
   - Gradle 4.6 or later
   - Java JDK 8 or later
+  - Check MarkLogic Version Compatibily [here](https://docs.marklogic.com/datahub/refs/version-compatibility.html)
```

Comment:
Should be "Compatibility". 

Also, style-wise, I think it's nicer to put "MarkLogic Version Compatibility" in brackets and have the link point to it as opposed to using the word "here".

Reply:
Okay sure. Will update it.